src/app/core/services/config/config.service.ts

```diff
@@ -130,6 +140,22 @@ export class ConfigService {
     })
   }
 
+  checkForAppUpdates() {
```

Comment:
How will this work for IOS? It will show up with a new Update prompt whenever we update to the playstore, right?

Reply:
Made the changes for iOS.